webpack/assets/javascripts/react_app/components/notifications/NotificationDrawerToggle.js

```diff
@@ -25,6 +26,8 @@ class NotificationDrawerToggle extends Component {
       case ACTIONS.RECEIVED_NOTIFICATIONS: {
         this.setState({
           count: NotificationsStore.getNotifications().length,
+          unreadCount: _.filter(NotificationsStore.getNotifications(),
+            notification => notification.seen).length,
           isLoaded: true
         });
         break;
```

Comment:
All the more reason not to do the calculations in `setState`. Kinda hard to set a break point.